tools/ci.psm1

```diff
@@ -462,7 +462,8 @@ function Invoke-CIFinish
                 $previewLabel= "daily{0}" -f $previewLabel
             }
 
-            $preReleaseVersion = "$previewPrefix-$previewLabel.$env:BUILD_BUILDID"
+            $prereleaseIteration = (Get-Random -Minimum 1 -Maximum 99)
```

Comment:
Perhaps rather than getting a random number, we could either peg this to `99` or use the last two digits of the `BUILD_ID`?

Reply:
Alternatively, we could put the iteration here in the 200+ range by taking the last two digits and prepending a 2? That's more work, but would definitively mark an asset as CI-generated

Reply:
My main concern is that adding randomness to builds isn't a good thing in terms of reproducibility

Reply:
From our perspective the build_ID is random.  

taking the last two digits would give things like 01, I'd have to cast that to an int, but that's possible.

The whole point here is to find these failures.  I don't think we should drop this.  If it becomes a problem, we can add a variable to override to be able to reproduce issues.

Reply:
> The whole point here is to find these failures

Which failures are you referring to?

> I don't think we should drop this

I didn't follow this part. Drop the randomisation? In my mind we're dropping information by assigning a random number. What's the scenario you have in mind here that this `Get-Random` call serves?

> If it becomes a problem, we can add a variable to override to be able to reproduce issues

I don't foresee it becoming a problem, but I think we should have good justification for introducing a random number vs something deterministic.

Reply:
> What's the scenario you have in mind here that this Get-Random call serves? 
This replicates our incrementing the pre-release iteration during preview.